qtpy/__init__.py

```diff
@@ -148,7 +162,9 @@ class PythonQtWarning(Warning):
 
             del macos_version
     except ImportError:
-        API = os.environ['QT_API'] = 'pyside2'
+        FALLBACK_API = 'pyside2'
+        warnings.warn(_fallback_fmt.format(API, FALLBACK_API), RuntimeWarning)
```

Comment:
I don't agree with displaying a warning for each binding because it'd make importing qtpy very verbose.

Instead, I think we should do this only if `_EXPLICIT_API is not None` and warn that a new binding `foo` is tried because `_EXPLICIT_API` couldn't be found.